locale/en/blog/vulnerability/http-server-security-vulnerability-please-upgrade-to-0-6-17.md

```diff
@@ -7,11 +7,12 @@ category: vulnerability
 slug: http-server-security-vulnerability-please-upgrade-to-0-6-17
 layout: blog-post.hbs
 ---
+
 ## tl;dr
 
 - A carefully crafted attack request can cause the contents of the HTTP parser's buffer to be appended to the attacking request's header, making it appear to come from the attacker.  Since it is generally safe to echo back contents of a request, this can allow an attacker to get an otherwise correctly designed server to divulge information about other requests.  It is theoretically possible that it could enable header-spoofing attacks, though such an attack has not been demonstrated.
 - Versions affected: All versions of the 0.5/0.6 branch prior to 0.6.17, and all versions of the 0.7 branch prior to 0.7.8.  Versions in the 0.4 branch are not affected.
-- Fix: Upgrade to [v0.6.17](http://blog.nodejs.org/2012/05/04/version-0-6-17-stable/, or apply the fix in [c9a231d](https://github.com/joyent/node/commit/c9a231d) to your system.
+- Fix: Upgrade to [v0.6.17](http://blog.nodejs.org/2012/05/04/version-0-6-17-stable/), or apply the fix in [c9a231d](https://github.com/joyent/node/commit/c9a231d) to your system.
```

Comment:
@nschonni weird this wasn't caught :/

Reply:
I think a bunch of the link checking is done through regex. Can't hurt to file an issue with the library though. Not sure if there is a rule that covers links or not